cfgov/unprocessed/css/pages/home.less

```diff
@@ -0,0 +1,93 @@
+.content_main__home {
+    padding-top: unit( @grid_gutter-width / @base-font-size-px, em );
+
+    .respond-to-max( @bp-xs-max, {
+        padding-bottom: unit( ( @grid_gutter-width * 2 ) / @base-font-size-px, em );
+    } );
+}
+
+// TODO: Create a base well for easier re-use in full width wells and FCM
+.o-well__full {
+    .respond-to-min( @bp-xl-min, {
+        // TODO: 45px should be moved to a global grid gutter var
+        // Max-width - both gutters: required due to well spanning the full page
+        max-width: 1230px - ( 45px * 2 );
+        margin: 0 auto;
+        padding-right: unit( 45px / @base-font-size-px, em );
+        padding-left: unit( 45px / @base-font-size-px, em );
+    } );
+}
+
+@home-hero_sm: 100%;
+@home-hero_med: 900px;
+@home-hero_lg: 1230px;
```

Comment:
I was purposely leaving them out so they're more generalized for exactly that reason. If I update the small one, I just want to change the value, and not have to update the var name as well.